Terminal.Gui/Views/TextField.cs

```diff
@@ -299,9 +299,9 @@ public override bool ProcessKey (KeyEvent kb)
 			// Needed for the Elmish Wrapper issue https://github.com/DieselMeister/Terminal.Gui.Elmish/issues/2
 			var oldCursorPos = point;
 
-			switch (kb.Key) {
+			switch (GetModifiersKey (kb)) {
 			case Key.DeleteChar:
-			case Key.ControlD:
+			case Key.ControlD | Key.CtrlMask:
```

Comment:
In this case I would suggest to change all the keys from `ControlA `to `ControlZ `into `A `to `Z `and in this case would be
 `case Key.D | Key.CtrlMask:`
What you think about this?

Reply:
Another thing which I not agree at all it's the `ControlSpace = 0` where is comment  "`The key code for the user pressing Control-spacebar`". But in my opinion isn't true because space is 32 and not 0. If I press Ctrl+Space I have a CtrlMask+32. I think the intencion was to represent the 0 (NULL Char) and the `ControlSpace `should be changed to "`Null`". The `Unknown `have to stay as is because is returned if no key is found. Of course this will force to change the default value from `ControlSpace `to `Null `instead . What do think about this?

Reply:
I always thought that `ControlsSpace = 0` looked wrong.